src/features/settings/components/RolesTable.tsx

```diff
@@ -0,0 +1,50 @@
+import { Box } from '@mui/system';
+import { FC } from 'react';
+import { Typography } from '@mui/material';
+import { DataGridPro, GridColDef } from '@mui/x-data-grid-pro';
+
+import { ZetkinOfficial } from 'utils/types/zetkin';
+
+type RolesTableProps = {
+  columns: GridColDef[];
+  description: string;
+  rows: ZetkinOfficial[] | undefined;
+  title: string;
+};
+
+const RolesTable: FC<RolesTableProps> = ({
+  columns,
+  description,
+  rows,
+  title,
+}) => {
+  return (
+    <>
+      <Box
+        sx={{
+          '& div': { backgroundColor: 'transparent' },
```

Comment:
I just wonder what this actually is..! I tried to spot the difference but my eyes can't find it 😵

Reply:
I will remove it because it's not applicable here, my bad!